src/SlackMessage.js

```diff
@@ -28,15 +28,15 @@ class SlackMessage extends SlackMessageRoot {
   constructor(token, message = { channel: '', ts: '' }) {
     super();
     if (!token) throw new Error('token is required');
-    this._token = token;
-    this._channel = message.channel;
-    this._ts = message.ts;
+    this.#_token = token;
+    this.#_channel = message.channel;
+    this.#_ts = message.ts;
```

Comment:
The only changes in this file are to use the hashtag-prefixed instance variables. It looks like the original author of the class intended to use JS private class features but omitted the `#` inside the actual class.